includes/adapter/shipment-options-from-order-adapter.php

```diff
@@ -48,6 +49,25 @@ private function isSignatureFromOptions(array $options, ?AbstractShipmentOptions
         return null;
     }
 
+    /**
+     * @param array                               $options
+     * @param AbstractShipmentOptionsAdapter|null $shipmentOptionsAdapter
+     *
+     * @return bool|null
+     */
+    private function isSameDayDeliveryFromOptions(array $options, ?AbstractShipmentOptionsAdapter $shipmentOptionsAdapter): ?bool
+    {
+        if (array_key_exists('same_day_delivery', $options)) {
+            return (bool) $options['same_day_delivery'];
+        }
+
+        if ($shipmentOptionsAdapter) {
+            return $shipmentOptionsAdapter->isSameDayDelivery();
+        }
+
+        return null;
```

Comment:
Waarom niet zoiets? Ik vind dit wat fijner omdat je dan in de return statement direct de logica/volgorde erachter ziet ipv dat je de functie moet "lezen" 😷 
```php
$valueFromOptions = $options['same_day_delivery'] ?? null;
$valueFromAdapter = $shipmentOptionsAdapter ? $shipmentOptionsAdapter->isSameDayDelivery() : null;

return $valueFromOptions ?? $valueFromAdapter;
```